packages/transformers/babel/src/BabelTransformer.js

```diff
@@ -64,16 +64,6 @@ export default (new Transformer({
       map.addIndexedMappings(rawMappings);
     }
 
-    let sourcesContent = await asset.getSourcesContent();
-    if (sourcesContent) {
-      for (let sourcesContentFileName of Object.keys(sourcesContent)) {
-        map.setSourceContent(
-          sourcesContentFileName,
-          sourcesContent[sourcesContentFileName],
-        );
-      }
-    }
-
     if (originalSourceMap) {
```

Comment:
`map.extends` should be removed in this transformer... this is probably a merge conflict that got resolved incorrectly. The AST should contain the mappings and the source contents should get set as Babels AST has no way to contain these

Reply:
Oh I thought extends would take care of setting all the source content?

Reply:
It does but it also remaps mappings which isn't necessary in this case as the babel ast should already contain the original mappings. So we just need to set source contents

Reply:
Ah. Ok well maybe we can still avoid an extra method in core by copying over the source contents from the original map here rather than extracting and storing it in core even when unnecessary.

Reply:
Do we have an integration test that covers this btw?

Reply:
Yes Will added a test for it, thought I also had one in the original PR but apparently not